packs/feats/wild-speech.json

```diff
@@ -31,7 +31,23 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "label": "Make an impression on animals",
```

Comment:
```suggestion
```
Either remove the label or localize it. I don't see the need to localize since we can actually automate this one.